tests/Zend/Loader/_files/ZendLoaderAutoloader/ZFAutoloadParseError.php

```diff
@@ -29,7 +29,7 @@
  * @copyright  Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
  * @license    http://framework.zend.com/license/new-bsd     New BSD License
  */
-class ParseError
```

Comment:
This existing ParseError class was in place for a deprecated test (removed below). I had to rename it to get it to work in my new tests, as the ParseError keyword is reserved in PHP7+.